configuration/config2/configfiles/hardware.xsd

```diff
@@ -0,0 +1,108 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<xs:schema
+    xmlns:xs="http://www.w3.org/2001/XMLSchema" elementFormDefault="qualified" attributeFormDefault="unqualified"
+    xmlns:hpcc="someuri">
+  <xs:include schemaLocation="types.xsd"/>
+  <xs:complexType name="hardware" hpcc:class="component" hpcc:category="" hpcc:displayName="Hardware Configuration">
+    <xs:sequence>
+      <xs:element name="Hardware" hpcc:docid="????">
+        <xs:annotation>
+          <xs:documentation>Section to describe the hardware that composes the environment.</xs:documentation>
```

Comment:
Have @JamesDeFabia look over the messages.

Reply:
My plan was for each component owner to go over the messages, tooltips, and any other documentation items. The owners should work with the doc team to get the correct wording. In addition, the doc team should take translations into account by collecting all of the types of locations where translatable text lives. For example, all hpcc:tooltip="text" and hpcc:displayName="text" occurrences.